composeApp/build.gradle.kts

```diff
@@ -6,7 +6,8 @@ plugins {
     alias(libs.plugins.composeMultiplatform)
     alias(libs.plugins.composeCompiler)
     alias(libs.plugins.spotless)
-    kotlin("plugin.serialization") version "2.2.0"
+
+    kotlin("plugin.serialization") version libs.versions.kotlin
```

Comment:
Couldn't this be an alias as wel?
`alias(libs.plugins.kotlinSerialization)`
With this in the toml? 
`kotlinSerialization = { id = "org.jetbrains.kotlin.plugin.serialization", version.ref = "kotlin" }`